mantidproject/mantid
# All usage of spectrum ids replaced with spectrum numbers in Engg Diffraction files

There are around 60 occurrences of `SpectrumIDs` or `SpecIDs` or `specIds` or `spec_ids` in `EnggDiffraction` files. As `SpectrumIDs` is not a concept in mantid any more, this pull requests replaces all usage of `spectrum IDs` with `spectrum Nos` from Engineering Diffraction GUI files.

**To test:**
1. Make sure no occurrence of `SpectrumIDs`, `SpecIDs`, `specIds` or `spec_ids` is found in any of the `EnggDiffraction` files
2. Check all GUI functionalities are still working, especially where `SpectrumNos` are utilised
3. Check `EnggDiffractionPresenterTest` still passes with any issues

Fixes #15529 

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
